AutomatedTesting/Gem/PythonTests/scripting/ScriptCanvas_TwoEntities_UseSimultaneously.py

```diff
@@ -40,50 +40,44 @@ def ScriptCanvas_TwoEntities_UseSimultaneously():
     """
 
     import os
-
-    import hydra_editor_utils as hydra
-    from utils import TestHelper as helper
-    from utils import Tracer
-
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.editor_component.editor_script_canvas import ScriptCanvasComponent
+    from editor_python_test_tools.utils import TestHelper
+    from editor_python_test_tools.utils import Tracer
     import azlmbr.legacy.general as general
     import azlmbr.math as math
-    import azlmbr.asset as asset
-    import azlmbr.bus as bus
 
-    LEVEL_NAME = "tmp_level"
+    TEST_ENTITY_NAME_1 = "test_entity_1"
+    TEST_ENTITY_NAME_2 = "test_entity_2"
     ASSET_PATH = os.path.join("scriptcanvas", "T92563191_test.scriptcanvas")
```

Comment:
Does this need the project root as part of the path?

Reply:
nope. not for this file. It's in automatedtesting/scriptcanvas